backend/server/server.go

```diff
@@ -15,16 +15,19 @@ import (
 	"github.com/rs/cors"
 )
 
+
 func Start() {
 	// CORS handler
-	c := cors.New(cors.Options{
-		AllowedOrigins: []string{
-			"http://localhost:8089",
-		},
-		AllowedMethods:   []string{"GET", "POST", "OPTIONS"},
-		AllowedHeaders:   []string{"Content-Type", "Authorization"},
-		AllowCredentials: true,
-	})
+        c := cors.New(cors.Options{
+        AllowedOrigins: []string{
+            "https://brierfoxforecast.com",
```

Comment:
We were able to host and configure with localhost:8089 only, having NGINX exposed on 8089.

Currently this is how http://brierfoxforecast.com/ is hosted.

Fewer Origins is better than more origins, more configurations.

Reply:
By hosting static files in nginx, these cors codes are not necessary.